daprdocs/content/en/developing-applications/local-development/multi-app-dapr-run/multi-app-overview.md

```diff
@@ -82,13 +85,94 @@ Logs for application and `daprd` are captured in separate files. These log files
 
 Even if you've decided to rename your resources folder to something other than `.dapr`, the log files are written only to the `.dapr/logs` folder (created in the application directory).
 
-
 ## Watch the demo
 
 Watch [this video for an overview on Multi-App Run](https://youtu.be/s1p9MNl4VGo?t=2456):
 
 <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/s1p9MNl4VGo?start=2456" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
+{{% /codetab %}}
+
+{{% codetab %}}
+<!--kubernetes-->
+
+## Multi-App Run template file
+
+Generate the multi-app template file (`dapr.yaml`) by running one of the following commands:
+
+```bash
+dapr run -k -f .
+```
+
+Or
+
+```bash
+dapr run -k -f dapr.yaml
+```
+
+The necessary default service and deployment definitions for Kubernetes are generated within the `.dapr/deploy` folder for each app in the `dapr.yaml` template. 
+
+If the `createService` field is present in the `dapr.yaml` template and set to `true`, then the `service.yaml` file(s):
+- Generate in the `.dapr/deploy` folder of each app in the multi-app run template
+- Are used to deploy the applications in a dev/test environment in K8s 
```

Comment:
```suggestion
If the `createService` field is set to `true` in the `dapr.yaml` template for an app, then the `service.yaml` file is generated in the `.dapr/deploy` folder of the app.
Otherwise only the `deployment.yaml` file is generated for each app that has the `containerImage` field set.
The files `service.yaml` and `deployment.yaml` are used to deploy the applications in `default` namespace in Kubernetes. This feature is specifically targeted only for running multiple apps in a dev/test environment in Kubernetes.
```